docker-compose.yml

```diff
@@ -1106,6 +1106,20 @@ services:
       - EX_APPS_NET=${EX_APPS_NET:-ipv4@localhost}
       - EX_APPS_COUNT=${EX_APPS_COUNT:-50}
 
+  appapi-dsp-harp:
+    # todo
+    image: nextcloud-appapi-harp:latest
+    container_name: nextcloud-appapi-harp
+    network_mode: ${NETWORK_MODE:-master_default}
```

Comment:
Any use case for switching the network mode? If just relevant for harp lets prefix the env variable